src/Skoruba.IdentityServer4.STS.Identity/Controllers/AccountController.cs

```diff
@@ -129,10 +131,30 @@ public async Task<IActionResult> Login(LoginInputModel model, string button)
 
             if (ModelState.IsValid)
             {
-                var user = await _userResolver.GetUserAsync(model.Username);
+                TUser user;
+                //TODO: Code smell.  Evaulate if this huge method can be refactored.
+                if (_isMultiTenant)
+                {
+                    user = await _userResolver.GetUserAsync(model.TenantCode, model.Username);
```

Comment:
probably bug here, right signature is 
`
public async Task<TUser> GetUserAsync(string login, string tenant = null)
`
so should be
`
    user = await _userResolver.GetUserAsync(model.Username, model.TenantCode); 
`